package_manifest.yaml

```diff
@@ -482,6 +482,9 @@ foreman_installer_packages:
       build_package_tito_builder: "tito.builder.FetchBuilder"
     katello-certs-tools: {}
     katello-selinux: {}
+    keycloak-httpd-client-install: {}
+    nodejs-node-gyp: {}
+    pcp-mmvstatsd: {}
```

Comment:
Where did these packages come from?

Reply:
what the hell, I have no idea. I'll remove the extras